src/openrct2/platform/Crash.cpp

```diff
@@ -320,7 +322,11 @@ static std::wstring GetDumpDirectory()
 {
     char userDirectory[MAX_PATH];
     platform_get_user_directory(userDirectory, nullptr, sizeof(userDirectory));
-    auto result = String::ToWideChar(userDirectory);
+
+    auto path = Path::Combine(userDirectory, "crashes");
```

Comment:
Our directory names are usually singular.

Reply:
Not quite but I changed.